pkg/wasm/types.go

```diff
@@ -83,7 +83,7 @@ const (
 	AuthServiceType      ServiceType = "auth"
 )
 
-// +kubebuilder:validation:Enum:=deny;allow
+// FailureModeType +kubebuilder:validation:Enum:=deny;allow
```

Comment:
This is breaking kubebuilder validation, isn't it?

Reply:
Actually, that kubebuilder annotation is useless, isn't it?

Reply:
It was there before, I just added the name for documentation completion on IDEs

Reply:
I think it is useless...  Anyway, let's not mix doc and annotations

```suggestion
// FailureModeType
+kubebuilder:validation:Enum:=deny;allow
```